volatility3/framework/plugins/windows/handles.py

```diff
@@ -243,7 +243,12 @@ def _make_handle_array(self, offset, level, depth=0):
         layer_object = self.context.layers[virtual]
         masked_offset = offset & layer_object.maximum_address
 
-        for entry in table:
+        for i in range(len(table)):
```

Comment:
Should the following be better?
```python
for index in enumerate(table):
```

Reply:
not sure if that will raise the exception on the same enumerate()

Reply:
I thought `enumerate` returned `index, value` not just `index`?

Reply:
I think it will raise the same exception, so we need to leave it as-is here:

```
 |  The enumerate object yields pairs containing a count (from start, which
 |  defaults to zero) and a value yielded by the iterable argument.
 |  
 |  enumerate is useful for obtaining an indexed list:
 |      (0, seq[0]), (1, seq[1]), (2, seq[2]), ...
 ```